### ISSUE TITLE
Audio attenuation by 20 dB with NDI

### ISSUE THREAD

**Issue by BlackScreen** [Palakis/obs-ndi#661]
Hello,
first of all: Thanks for this great plugin.

One little issue we have:
The audio output in NDI, when send via NDI in OBS, is 20 dB too low.
Example: A video with a sine tone (1 kHz) at -18 dBFS is running in OBS.
When output over NDI, I recorded it in the NDI Studio Monitor, the peak of the sine tone is -38 dBFS, so 20 dB too low.

I have come accross a similar issue in a forum over at vMix, that describes the problematic pretty good.
https://forums.vmix.com/posts/t18413-NewTek-Spark-Audio-Noise

vMix has fixed it like this:
"As a postscript to all this, the option of a 20dB adjustment to incoming and outgoing NDI audio levels was added to vMix in v23. For outputs, it can be found in the Output Settings for each output in the "Outputs/NDI/SRT" tab of the main Settings dialog window (shown as a "20dB Boost" checkbox), and for inputs it's in the General tab of the Audio Settings for individual NDI inputs (shown as a "-20dB Pad" checkbox)."

Maybe we could have something like that in OBS as well? Would be a good solution, as long as audio is handled this way in NDI.
Thank you for considering! :)

**Comment by BlackScreen** [comment]
Have you had time to look into this issue?

**Comment by paulpv** [comment]
Can you just add a Compressor filter with a Gain to correct the problem?

**Comment by BlackScreen** [comment]
> Can you just add a Compressor filter with a Gain to correct the problem?

No. This would be a workaround, but not a solution for the long term.

**Comment by Trouffman** [comment]
This is technically "not" a bug but this could be improved.

Reason why some feed seems "20db too low" is because of the difference between SMPTE & EBU level.

More to read on : https://docs.ndi.video/all/developing-with-ndi/sdk/frame-types#audio-frames-ndilib_audio_frame_v3_t

For Version 6.0.0 and prior, there was mutliple version of the audio_frame in use. Starting 6.1.0 are exclusively using audio_frame_v3 and might help int he future to have options to have SMPTE or EBU reference level.
